crates/apollo-router/src/router_factory.rs

```diff
@@ -19,23 +19,41 @@ where
         &self,
         configuration: &Configuration,
         schema: Arc<graphql::Schema>,
+        plan_cache_limit: usize,
     ) -> future::BoxFuture<'static, Router>;
+    fn recreate(
+        &self,
+        graph: Arc<Router>,
```

Comment:
This thing has been renamed  `router`.
```suggestion
        router: Arc<Router>,
```

Reply:
done